packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -207,7 +207,7 @@ class ClayMultiSelect extends ClayComponent {
 	 * @return {Boolean} If the event has been prevented or not.
 	 */
 	_handleOnInput(event) {
-		const value = event.data.value;
+		const value = event.data.value.replace(',', '');
```

Comment:
What will this do if I do something like:

* Type "foobar", then move the cursor left three times and type ","? (I'm expecting it will appear to swallow the keypress)
* Paste a string like "a,b,c"? (I'm expecting it will blow away only the first comma)

I'm thinking that maybe a stricter check might be required.

Reply:
hey @wincent, thanks for your concerns with these cases, really they will do what you expect. Do you have any suggestions about this?

Reply:
So, what should happen if someone copy-pastes something like `a`, or `a,b` or `a , b , something,`?

Reply:
So, shouldn't we autocreate tags on pasting? I was under the impression that that would be a preferred approach.

Reply:
Talking with David A. we show that is better to have the functionality that detects all commas and creates at tag per element before a comma.
In this way:

- a , b , something, ---> [a][b][something] please scape white spaces when after the last char previous to comma
- a , b , something ---> [a][b]something

Reply:
Sounds good @victorvalle: as a user I'd probably find that the most pleasant experience. 🎉